src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -106,4 +107,132 @@ public static boolean isWater(IBlockState iBlockState)
         return Objects.equals(iBlockState, Blocks.water.getDefaultState())
                || Objects.equals(iBlockState, Blocks.flowing_water.getDefaultState());
     }
+
+    /**
+     * Gets an item from a block.
+     * @param block input block.
+     * @return output item.
+     */
+    public static Item getItemFromBlock(Block block)
+    {
+        if(block instanceof BlockBanner)
+        {
+            return Items.banner;
+        }
+        else if(block instanceof BlockBed)
+        {
+            return Items.bed;
+        }
+        else if(block instanceof BlockBrewingStand)
+        {
+            return Items.brewing_stand;
+        }
+        else if(block instanceof BlockCake)
+        {
+            return Items.cake;
+        }
+        else if(block instanceof BlockCauldron)
+        {
+            return Items.cauldron;
+        }
+        else if(block instanceof BlockCocoa)
+        {
+            return Items.dye;
+        }
+        else if(block instanceof BlockCrops)
+        {
+            if(block instanceof BlockCarrot)
+            {
+                return Items.carrot;
+            }
+            else if(block instanceof BlockPotato)
+            {
+                return Items.potato;
+            }
+            return Items.wheat_seeds;
+        }
+        else if(block instanceof BlockDaylightDetector)
+        {
+            return Item.getItemFromBlock(Blocks.daylight_detector);
+        }
+        else if(block instanceof BlockDoor)
+        {
+            return block == Blocks.iron_door ? Items.iron_door : (block == Blocks.spruce_door ? Items.spruce_door :
+                                                                  (block == Blocks.birch_door ? Items.birch_door :
+                                                                   (block == Blocks.jungle_door ? Items.jungle_door :
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ':' should be on a new line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AOperator_Wrap)